src/components/Nav.jsx

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
 import { Route, Routes, Link } from 'react-router-dom';
-import Books from './Books';
 import Categories from './Categories';
+import Ui from './Ui';
 
 function Nav() {
   return (
     <>
       <header className="head">
         <h1>Bookstore CMS</h1>
         <nav className="navigation">
-          <Link to="/" className="link">books</Link>
+          <Link to="Ui" className="link">books</Link>
           <Link to="/Categories" className="link">Categories</Link>
         </nav>
       </header>
       <Routes>
-        <Route path="/" element={<Books />} />
+        <Route path="Ui" element={<Ui />} />
         <Route path="/Categories" element={<Categories />} />
       </Routes>
```

Comment:
- [ ] In keeping with convention, routing should not be defined in the Navbar component. Instead, it should be in the App component. Kindly move this code from here to the App component. Under the `<Nav />` link